src/overlays/actors/ovl_En_Dai/z_en_dai.c

```diff
@@ -441,9 +457,9 @@ void func_80B3F00C(EnDai* this, PlayState* play) {
     Math_ApproachS(&this->actor.shape.rot.y, this->actor.world.rot.y, 3, 0x2AA8);
 }
 
-void func_80B3F044(EnDai* this, PlayState* play) {
-    static s32 D_80B3FE38[] = {
-        0, 0, 6, 7, 8,
+void EnDai_HandleCutscene(EnDai* this, PlayState* play) {
```

Comment:
Part of me is thinking to name these pure Cutscene functions as `_HandleCutscene` when it's an `actionFunc` and `_UpdateCutscene` when it's just called directly from `_Update(`, thoughts?

Reply:
I think I'm slightly more inclined to just name them all `HandleCutscene` since they do the same thing, just slightly different implementation details.